extensions/amp-consent/0.1/consent-config.js

```diff
@@ -244,6 +246,25 @@ export class ConsentConfig {
     return mergedConfig;
   }
 
+  /**
+   * Validate if story consent then no promptUiSrc
+   * @param {!JsonObject} config
+   * @return {!JsonObject}
+   */
+  checkStoryConsent_(config) {
+    const isStoryConsent = !!childElementByTag(
+      this.element_,
+      AMP_STORY_CONSENT_TAG
+    );
+    userAssert(
+      !config.promptUISrc || (config.promptUISrc && !isStoryConsent),
```

Comment:
The config comes from an inline json object, use quoted string to refer to property. https://developers.google.com/closure/compiler/docs/limitations#implications-of-global-variable,-function,-and-property-renaming:

Reply:
Thanks for the reference. Will be sure to use this in the future.